include/seqan3/argument_parser/validators.hpp

```diff
@@ -85,7 +85,7 @@ class integral_range_validator;
  * \details
  *
  * On construction, the validator must receive a maximum and a minimum number.
- * The struct than acts as a functor, that throws a seqan3::parser_invalid_argument
+ * The struct than acts as a functor, that throws a seqan3::invalid_argument
```

Comment:
Could you change all exceptions that are thrown in this file by validators into `validation_failed` exceptions? :pray:  This would be much for consistent that what I had before.